src/main/java/frc/robot/Robot.java

```diff
@@ -175,19 +178,11 @@ public void setIndexerSpeed(final double indexerSpeed) {
   @Config(tabName = "Autonomous", name = "Distance", defaultValueNumeric = 36)
   public void setAutonStraightDistance(final double distance) {
     this.distance = distance;
-    this.autonomousCommand.setDistance(this.distance);
   }
 
   @Config(tabName = "Autonomous", name = "Maximum Speed", defaultValueNumeric = .75)
   public void setAutonMaxSpeedForDriveStraight(final double maxSpeed) {
     this.maxSpeed = maxSpeed;
-    this.autonomousCommand.setMaxSpeed(this.maxSpeed);
```

Comment:
need to add this back to autonomousCommandGroup to pass down.  Or delete the method entirely.